app/helpers/post_helper.rb

```diff
@@ -1,6 +1,6 @@
 module PostHelper
   def author_links(post, linked: true, colored: false)
-    total = post.authors.size
+    total = post.authors.length
```

Comment:
(Relation no longer has size?)

Reply:
I think it has size but size does something super weird on this method in particular? I remember a whole adventure finding that bug

Reply:
https://stackoverflow.com/questions/6083219/activerecord-size-vs-count size tries to be intelligent about "use array size in memory if queryset is loaded, call db for count if not" but if our db and memory counts differ that could be the problem? If not that I am short term out of ideas.

Reply:
Reloading the posts in tests before calling this seems to fix it, yeah! Little concerned about why this is a problem with the upgrade in case it affects anything else, but.

Reply:
Weird! We investigated this in the Discord and couldn't figure out exactly what's going on, but it seems like a potential Rails bug, and doesn't seem to impact our use cases.